skema/program_analysis/CAST/python/ts2cast.py

```diff
@@ -148,7 +156,29 @@ def visit_assignment(self, node: Node) -> Assignment:
             right=self.visit(right),
             source_refs=[literal_source_ref]
         )
+
+    def visit_unary_op(self, node: Node) -> Operator:
+        """
+            Unary Ops
+            OP operand
+            where operand is some kind of expression
+        """
+        op = self.node_helper.get_operator(node.children[0])
```

Comment:
It likely doesn't matter in this case, but be careful when accessing the children of a node by index. There are cases where the structure may change.

For example, a comment node can shift all of the children indexes by 1. 

This was one of the motivations for the various helper functions in node_helper.py.

Reply:
Oh, that's super important to know actually. That makes a lot of sense. I think as part of the next feature I port over I'll add helper functions like you have in order to defend against this, because that will most definitely come up somewhere. Thank you for that!